api/public/js/main.js

```diff
@@ -0,0 +1,9 @@
+function deleteAction (endpoint, token) {
```

Comment:
Could you enlighten me why this is used when deleting? Is it to not show an id, not allow other users to try and delete it or something? This could then go to app.js instead of a public folder, or is that not feasible?

Reply:
Iskeliau trinima i js funckiaj kad nereiktu hidden formu kurti o paskui jas submitinti.

Reply:
no fix needed, app.js is better not used for modifactions beaces it has prewriten code

Reply:
Fair point for hidden forms. But I still don't like it being in the public folder. 
As far as I know, app.js has only `require('./bootstrap');`. You're looking at public/js/app.js. Laravel stores pre-compiled javascript files (the files you write code in) in resources/js.

Reply:
oh right, will fix but probs not today